firmware/config/engines/honda_k_dbc.cpp

```diff
@@ -42,8 +42,7 @@ void setHondaK() {
 	engineConfiguration->injectionMode = IM_SEQUENTIAL;
 
 	engineConfiguration->crankingIACposition = 70;
-    engineConfiguration->postCrankingFactor = 1.25;
```

Comment:
would setTable(engineConfiguration->postCrankingFactor, 1.25f); be a more compatible change here?

Reply:
done, that one was ending at different time, re-scaled bins for that